Dialog/core/designsystem/src/commonMain/kotlin/com/on/dialog/designsystem/component/button/Button.kt

```diff
@@ -0,0 +1,202 @@
+package com.on.dialog.designsystem.component.button
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.RowScope
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.sizeIn
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.OutlinedButton
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.on.dialog.designsystem.icon.DialogIcons
+import com.on.dialog.designsystem.theme.DialogTheme
+import org.jetbrains.compose.ui.tooling.preview.Preview
+
+@Composable
+fun DialogButton(
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    enabled: Boolean = true,
+    contentPadding: PaddingValues = ButtonDefaults.ContentPadding,
+    shape: Shape = RoundedCornerShape(DialogButtonDefaults.Default.cornerRadius),
```

Comment:
```suggestion
    shape: Shape = DialogButtonDefaults.shape,
```
object로 바꾼 default면 다음과 같은 방식으로 변경될 수 있을 거 같습니다.😊